src/tree/branch.rs

```diff
@@ -18,24 +18,25 @@ use nstack::NStack;
 /// Represents a level of a branch on a given depth
 #[derive(Debug, Default, Clone, Copy, Canon)]
 pub struct PoseidonLevel {
-    level: [BlsScalar; dusk_hades::WIDTH],
+    pub(crate) level: [BlsScalar; dusk_hades::WIDTH],
```

Comment:
Since the tests have been pulled to unit level, does this need to be `pub(crate)`?

Reply:
Yes, otherwise the tests can not alter the levels to create a faulty branch.